client/mds/infra-connector/service.go

```diff
@@ -176,3 +193,99 @@ func (s *Service) DeleteDataPlane(id string) error {
 
 	return nil
 }
+
+
+func (s *Service) CreateCloudAccount(requestBody *CloudAccountCreateRequest) (*model.MdsCloudAccount, error) {
+	if requestBody == nil {
+		return nil, fmt.Errorf("requestBody cannot be nil")
+	}
+	var response model.MdsCloudAccount
+	urlPath := fmt.Sprintf("%s/%s", s.Endpoint, CloudAccount)
+
+	_, err := s.Api.Post(&urlPath, requestBody, &response)
+	if err != nil {
+		return &response, err
+	}
+
+	return &response, err
+}
+
+// UpdateCloudAccount - To Update the cloud account
+func (s *Service) UpdateCloudAccount(id string, requestBody *CredentialModel) error {
+
+	urlPath := fmt.Sprintf("%s/%s/%s", s.Endpoint, CloudAccount, id)
+	_, err := s.Api.Put(&urlPath, requestBody, nil)
+
+	if err != nil {
+		return err
+	}
+
+	return err
+}
+
+// DeleteCloudAccount - Submits a request to delete cloud account
+func (s *Service) DeleteCloudAccount(id string) error {
+	urlPath := fmt.Sprintf("%s/%s/%s", s.Endpoint, CloudAccount, id)
+
+	_, err := s.Api.Delete(&urlPath, nil, nil)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (s *Service) CreateCertificate(requestBody *CertificateCreateRequest) (*model.MdsCertificate, error) {
+	if requestBody == nil {
+		return nil, fmt.Errorf("requestBody cannot be nil")
+	}
+	var response model.MdsCertificate
+	urlPath := fmt.Sprintf("%s/%s", s.Endpoint, Certificate)
+
+	_, err := s.Api.Post(&urlPath, requestBody, &response)
+	if err != nil {
+		return &response, err
+	}
+
+	return &response, err
+}
+
+func (s *Service) UpdateCertificate(id string, requestBody *CertificateUpdateRequest) (*model.MdsCertificate, error) {
+	if requestBody == nil {
+		return nil, fmt.Errorf("requestBody cannot be nil")
+	}
+	var response model.MdsCertificate
+	urlPath := fmt.Sprintf("%s/%s/%s", s.Endpoint, Certificate, id)
+
+	_, err := s.Api.Post(&urlPath, requestBody, &response)
+	if err != nil {
+		return &response, err
+	}
+
+	return &response, err
+}
+
+func (s *Service) GetCertificate(id string) (model.MdsCertificate, error) {
+	var response model.MdsCertificate
+
+	reqUrl := fmt.Sprintf("%s/%s/%s", s.Endpoint, Certificate, id)
+
+	_, err := s.Api.Get(&reqUrl, nil, &response)
+	if err != nil {
+		return response, err
+	}
+	return response, nil
```

Comment:
it can be `return response, err`, similarly can be done in other created methods